account_banking_payment_export/wizard/payment_order_create.py

```diff
@@ -56,8 +75,7 @@ def extend_payment_order_domain(self, payment_order, domain):
             # Do not propose partially reconciled credit lines,
             # as they are deducted from a customer invoice, and
             # will not be refunded with a payment.
-            domain += [('credit', '>', 0),
-                       '|',
+            domain += ['|',
```

Comment:
I just made a test, and I really don't understand why credit > 0 is removed from the domain.

For instance, I have a supplier with a credit of 15 and a debit of 5, partially reconciled together.

The payment order wizard proposes both, which is weird in itself, and if you select both, you get two payment lines both with an amount of 10 (which is the residual).